src/Chainweb/BlockHeader.hs

```diff
@@ -710,6 +766,10 @@ _blockAdjacentChainIds =
 blockAdjacentChainIds :: Getter BlockHeader (HS.HashSet ChainId)
 blockAdjacentChainIds = to _blockAdjacentChainIds
 
+-- | @getAdjacentHash cid h@ returns the adjacent hash of h for chain cid. It
+-- throws a @ChainNotAdjacentException@ if @cid@ is not adajcent with @_chainId
+-- h@ in the chain graph of @h@.
```

Comment:
Haddocks quesstion: is `@h@` the same as `'h'`? If so I find the latter much more readable in code ...

Reply:
`@...@` and `'...'` have different meaning in Haddock. `'...'` is for references to functions defined in the code , for which a hyper link is created. `@...@` is for arbitrary text that is formatted verbatim. 

I think, recent versions of Haddock all support markdown style back-tick quotation for verbatim text, but I haven't tried that yet and have been sticking to the backward compatible `@...@` notation. Maybe it's time to try  out back-tick quotes and switch to that syntax if it works for our builds?